src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -403,6 +411,7 @@ public FeedVersionCommentActivity(Note note, FeedSource feedSource, FeedVersion
     }
 
     public static void register (String apiPrefix) {
+        get(apiPrefix + "secure/user", UserController::getUserProfileFromToken, json::write);
```

Comment:
This conflicts with an existing endpoint and is causing the User Management in the UI to not work.

Reply:
Yea, I think this endpoint is unnecessary anyways. I'm reverting this commit.